runtime/src/main/java/org/corfudb/runtime/collections/Table.java

```diff
@@ -322,6 +325,57 @@ public void preCommitCallback(TokenResponse tokenResponse) {
         return keyOfQueueEntry;
     }
 
+    /**
+     * Helper function for logUpdateThatLooksLikeEnqueue() in TxnContext.java
+     * @param e the element to update
+     * @throws IllegalArgumentException if some property of the specified
+     *                                  element prevents it from being updated
+     */
+    public K logUpdateEnqueue(V e, TxnContext txn, CorfuStore corfuStore) {
```

Comment:
we dont need `txn` or `corfustore` right? TxnContext is only used as a wrapper for the AbstractTransctionalContext()'s logUpdate and corfustore is only used to get the protobuf serializer code which can be a public static right?
Also we need the stream tags